### DISCUSSION TITLE
How to start contributing to spacedrive?

### DISCUSSION THREAD

**Discussion by yaten2302** [spacedriveapp/spacedrive#2775] [Community Support] [answered]
Hey @iLynxcat , 
I'm Yaten, I'm an active open source contributor and I've knowledge of JavaScript, TypeScript, ReatJS and currently learning Golang. I'm also contributing to Meshery(a CNCF project). I want to start to start my contribution with Spacedrive. Could you please guide that the open issues which are there, can they be taken up by OSS contributors or are those limited to the Spacedrive team? I've good knowledge of TypeScript and I can contribute to the frontend part of the project.

Thanks

**Comment by HeavenVolkoff** [comment, marked as answer]
Hi @yaten2302, thank you for your interest in contributing to our project! Our [contributing guide](https://github.com/spacedriveapp/spacedrive/blob/main/CONTRIBUTING.md) should provide all the instructions you need to set up a development environment and start making changes or fixing bugs in Spacedrive. As for the issues, you are welcome to work on most of them. We just ask that you express your interest in the issue itself with a message so we’re aware and can avoid any overlap, unnecessary rework, or potential frustration with rejected PRs. Also, any issues labeled with `[(ENG|MOB)-<number>]` are tracked in our private issue tracker and likely already have someone assigned to it. For those, we’ll need to check whether they’re being worked on and if it’s possible to hand them off to external contributors.
